### ISSUE TITLE
Question: runClient

### ISSUE THREAD

**Issue by ntmleafia** [Alcatergit/Hbm-s-Nuclear-Tech-GIT#432]
i'm using IntelliJ IDEA, when starting off fresh project with forge's mdk, it comes with runClient feature on gradle that starts up minecraft for me, but that doesn't work with this repository.
Tried everything I could do, but since I'm an absolute java noob I don't even have any idea what's going on.

Anyone know how to fix runClient for NTM Extended
Everytime it just says "Task '[task name here]' not found in root project", not limited to runClient, but tasks like build do work fine

**Comment by TheSlize** [comment]
I personally use build then launch Minecraft with the mod by myself.

**Comment by ntmleafia** [comment]
yeah i did that billion times and... i wish that could be made better

**Comment by anonymousomeone** [comment]
to fix task not found you can disable configureondemand in gradle.properties but then it will crash on loading with
`Caused by: java.io.FileNotFoundException: hbm:models/error.obj`
